daemon/src/db.rs

```diff
@@ -350,6 +549,393 @@ impl Connection {
     }
 }
 
+async fn load_collaborative_settlement_tx(
```

Comment:
You called the table `collaborative_close_txs`, might be a good idea to align the naming. i.e. `load_collaborative_close_tx` or the otherway around.

Reply:
Fair point. I did not achieve consistent naming throughout. I'll align it.